### ISSUE TITLE
Depends on RapidApp::Util, but is also depended on by RapidApp

### ISSUE THREAD

**Issue by jwrightecs** [vanstyn/DBIx-Class-Schema-Diff#6]
```
==> Found dependencies: RapidApp::Util
--> Working on RapidApp::Util
Fetching http://www.cpan.org/authors/id/V/VA/VANSTYN/RapidApp-1.3302.tar.gz ... OK
Configuring RapidApp-1.3302 ... OK
==> Found dependencies: DBIx::Class::Schema::Diff
! Installing the dependencies failed: Module 'DBIx::Class::Schema::Diff' is not installed
! Bailing out the installation for RapidApp-1.3302.
! Installing the dependencies failed: Module 'RapidApp::Util' is not installed
! Bailing out the installation for DBIx-Class-Schema-Diff-1.11.

```
if I remove the use RapdApp::Util from lib/DBIx/Class/Schema/Diff/State.pm and remove RapidApp::Util from requirements in META.yml and Makefile.PL, I can build and install this module

**Comment by jwrightecs** [comment]
#7

**Comment by vanstyn** [comment]
thanks for catching that. Just shipped 1.12 to CPAN with fix in #7
